examples/inverse/plot_resolution_metrics_eegmeg.py

```diff
@@ -0,0 +1,135 @@
+"""
+==============================================================
+Compute spatial resolution metrics to compare MEG with EEGMEG.
+==============================================================
+
+Compute peak localisation error and spatial deviation for the point-spread
+functions of dSPM and MNE. Plot their distributions and difference
+distributions.
+This example mimics some results from [1]_, namely Figure 3 (peak localisation
+error for PSFs, L2-MNE vs dSPM) and Figure 4 (spatial deviation for PSFs,
+L2-MNE vs dSPM).
+
+References
+----------
+.. [1] Hauk et al. "Towards an Objective Evaluation of EEG/MEG Source
+Estimation Methods: The Linear Tool Kit", bioRxiv 2019,
```

Comment:
fix indent. Basically `.. ` says it should be indented for rst style formatting

Reply:
I now indented as in your comment above, but I don't know about "rst style formatting".